src/components/reusuables/YohDevButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import { Button } from "react-bootstrap"
+import { Link } from "gatsby"
+
+const YohDevButton = (props) => {
```

Comment:
Destructure props like the last review.
(props) should be ({heading, subtitle, etc})